src/dbnode/storage/bootstrap/process.go

```diff
@@ -372,3 +376,66 @@ func (b bootstrapProcess) newRunOptions() RunOptions {
 		).
 		SetInitialTopologyState(b.initialTopologyState)
 }
+
+// NewNamespaces returns a new set of bootstrappable namespaces.
+func NewNamespaces(
+	namespaces []ProcessNamespace,
+) Namespaces {
+	namespacesMap := NewNamespacesMap(NamespacesMapOptions{})
```

Comment:
Can probably init this with size yeah?
```
	namespacesMap := NewNamespacesMap(NamespacesMapOptions{
		InitialSize: len(namespaces),
	})
```

Reply:
Can't be bothered when it's in such a low frequency call code path TBH.